### DISCUSSION TITLE
Can anybody give me some advices about my Portfolio? ❤️

### DISCUSSION THREAD

**Discussion by gaomingzhao666** [community/community#131218] [Programming Help]
I build a portfolio to start my career in October 2025

My career probably will start in Japan, and this portfolio has three translations which include `English` `Mandarin` `And Japanese`.

I need some bits of advice on this portfolio and some criticisms about the pros and cons of it.

Here is the link to that [**Portfolio**](https://nano-portfolio-ruby.vercel.app/)

Thanks, everybody. 😊

**Comment by BryanGabrielAtangana** [comment]
Hey,

I took a quick look at your portfolio and I really like the light version. It's minimalist and original. You might want to work more on the animations to make them seamless and smooth. For animations, you could consider using libraries like GSAP or Framer Motion.

I noticed that when you change the language, it updates on the site but not on the select list. You might want to synchronize the select list with the current language setting for better user experience.

There's still some work to be done on the dark mode appearance. You could also consider using a lighter shade for dark mode for better readability. 

Overall, great job! I hope the final version turns out spot on.

**Reply by gaomingzhao666** [reply]
Thank you so much for your opinions, bro.

I just used default configuration support by the Nuxt.js team, there are still a lot of things I didn't know and I will continue to keep learning, your criticism is pretty useful for me and thanks again bro. 😊

**Comment by 4darsh-Dev** [comment]
@gaomingzhao666
your portfolio looks great, and the best part was comment section, 
you can add certificates/achievements section this will boost your portfolio, providing proof of work, 
Also adding meta keywords and opengraph (og) tags for socials will improve ranking of website
But I don't know why anyone will login to your portfolio, 
My portfolio : https://adarshmaurya.onionreads.com/

**Reply by gaomingzhao666** [reply]
Thanks for your advices bro, Your portfolio and your works looks so great and responsive.

I build the login/register functionalities because I just learned mongoose/express/node.js in backend section and I want to build functionalities to practice that skills I just learned, in this case, user must be login can post and like comments.

I am just a freshman so there has a lot of things I still to learn, I am so appreciate your advices thanks bro. 😄

**Comment by GabrielLins64** [comment]
First of all, I would like to commend some points that I really liked:

- Dark/light theme toggle. I simply love platforms that offer this;
- Responsiveness (adaptation to different screen sizes);
- Accessibility by providing different possible languages;
- And congratulations on completing your bachelor's degree!

I have analyzed some challenges and portfolios as a recruiter, and I will give you some suggestions on what I believe you could work on (even though it's already very good). These are just minor details, opinions, and personal preferences, but they might make some difference:

- In light mode, I would add just a bit more box-shadow on some sides of the cards because, on some screens, it might seem like there is no card due to the box-shadow only appearing at the bottom and very faintly. (When applying it, increase the screen brightness to try to imagine how an observer sees the card).

![image](https://github.com/community/community/assets/42005625/7e70cf67-f246-4f33-ad6c-e20f386aadc3)

- On mobile devices, if the user turns the device sideways, the index cards are aligned to the left, which looks a bit odd visually. Personally, I would place them in a centered container on screens smaller than tablets.

![image](https://github.com/community/community/assets/42005625/3ef38a7f-775d-4f9a-b60f-4fa11dc59bdc)

- Again on mobile devices, if the user turns the device sideways (hehehe sorry for being picky and detailed), the text on the "career" screen has no vertical scroll and seems to have a hardcoded top margin of 13rem, which aligns the text at the bottom. It might be interesting to look for ways to achieve some vertical centering, or set the top margin based on the device's height (which is automatically adjusted when we turn the device sideways).

![image](https://github.com/community/community/assets/42005625/a3fb9343-c9e0-40b1-b573-2e9a49841be9)

- Depending on the screen width, the "windows" button on the comments card sticks out. It might be a good idea to consider using a grid within this card.

![image](https://github.com/community/community/assets/42005625/12ee27f5-7cc6-4b20-8562-87fce7c62dce)

- This one is not about technical details, it's just a personal opinion, and feel free to ignore it completely: I would opt for an email that looks a bit more professional than "gaomingzhao666@outlook.com", as this might negatively impact depending on the recruiter evaluating.

**Comment by gaomingzhao666** [comment]
I really pleasure to thank you for your detailed advices, and some those of issue have been solved depends on your advices. ❤️

I am confused about the issue on the `comment card` component, I have never used `grid layout` on the component like this, I always use `grid layout` on `video card` like YouTube home page or such `comment card` that so apparent two dimension layout, and I always use `flex` layout inside of those card component, I think I need to hide or short calendar string and hidden or just display an OS icon on sm size of the screen, this is I always do, it's sounds like so stupid than grid solution but I am just stupid so I always think ugly solution.😂   

The email address is my true name and we can not use like `Google` `GitHub` `Amazon AWS` and other international apps and websites, so only I can use inside of China mainland is `MicroSoft` products, personally I am more prefer to use Gmail.

**Reply by GabrielLins64** [reply]
The grid was only a suggestion that I personally like to use in responsive layouts, although flex is also a very good approach. But as you said, the label coming out of the card can be solved in several ways, nothing that actually solves the problem is an ugly solution. Regarding the email thing, I have nothing against your name xD (sorry I didn't mean to sound like that), I was referring to the "666" part, which may be a bit radical symbolism for some evaluators.

**Reply by gaomingzhao666** [reply]
Thanks bro.

In my opinion, I always want to find the best way to handle a specific issue, so I am so confused when I must make a decision between several solutions. So I said above that grid is more elegant method to handle this question but I can not use it well in `in-box` component.

About `666`, actually it means `good good good` in my mother language. hahaha😂

**Comment by gaomingzhao666** [comment]
Have other guys review my portfolio. 🥺

**Comment by cavalluccijohann** [comment]
Hey,

Your portfolio is pretty good! Maybe you can add a contact form (check out Resend; it's very good and easy for this). Also, on mobile, you can't post comments. I'm not sure if that's intentional. ☀️

**Reply by gaomingzhao666** [reply]
Hey bro, thank you so much for your advices for my Portfolio.

I will add a form by using `Resend` in next commit, I never heard this lib before and it looks awesome, thanks for your recommendation. 🥰

**Comment by sgfrdgrln** [comment]
It's a cool and unique portfolio, it is my first time seeing a portfolio that requires the user to login! and it's nice that you can commit in it. Good job!

**Reply by gaomingzhao666** [reply]
Thank you so much for your response for my portfolio and thanks that compliment.🥰

**Comment by f9ki3** [comment]
Hello @gaomingzhao666 . Your portfolio look really good. Minimalist and you have an eye on design you know how to handle which composition you want to put in it.  From navigation, sidebar panel, and content of each pages. I believe it will make it more popup if u put an image or some graphics in each of your cards in my opinion. Over all I like it

**Reply by gaomingzhao666** [reply]
Thank you so much for your compliment and great advice!! and thanks you clicked so many hearts on my comment on the portfolio!!❤

I will improve it depends on your advices.
